roles/default/kiali-deploy/defaults/main.yml

```diff
@@ -97,6 +97,9 @@ kiali_defaults:
     grafana:
       auth:
         ca_file: ""
+        component_status:
+          app_label: "grafana"
+          is_core: false
```

Comment:
There should be a corresponding commented section at around line 484 in kiali_cr.yaml to document these defaults.